geoips/commandline/geoips_tree.py

```diff
@@ -0,0 +1,195 @@
+"""GeoIPS CLI "tree" command.
+
+Single 'geoips tree' command which will display GeoIPS CLI commands up to a --max_depth
+value.
+"""
+
+from colorama import Fore, Style
+
+from geoips.commandline.geoips_command import GeoipsExecutableCommand
+
+
+class GeoipsTree(GeoipsExecutableCommand):
+    """Tree Command Class for displaying GeoIPS CLI commands in a tree-like fashion.
+
+    Can be configured to print in a colored fashion, up to a maximum depth, and whether
+    or not we want the full command string or just the command name at it's depth.
+    """
+
+    name = "tree"
+    command_classes = []
+
+    @property
+    def top_level_parser(self):
+        """The parser associated with the top level 'geoips' command."""
+        if not hasattr(self, "_top_level_parser"):
+            # Get 'geoips' parser
+            self._top_level_parser = self.parent.parser
+        return self._top_level_parser
+
+    @property
+    def level_to_color(self):
+        """A color-mapping dictionary based on the depth of the command tree."""
+        if not hasattr(self, "_level_to_color"):
+            self._level_to_color = {
+                0: Fore.RED,
+                1: Fore.YELLOW,
+                2: Fore.BLUE,
+                3: Fore.MAGENTA,
+            }
+        return self._level_to_color
+
+    @property
+    def cmd_line_url(self):
+        """The url to the GeoIPS documentation for the GeoIPS CLI."""
+        if not hasattr(self, "_cmd_line_url"):
+            self._cmd_line_url = r"https://nrlmmd-geoips.github.io/geoips/userguide/command_line.html"  # NOQA
+        return self._cmd_line_url
```

Comment:
Should this be set somewhere else besides in this command line script?  An environment variable -> base_paths.py which will eventually be in a config ?

Reply:
Oh yeah, for now we can probably set 

gpaths["GEOIPS_DOCS_URL"] or something?  In base_paths?

Reply:
I'll make that change.